drivers/sensor/ccs811/ccs811_trigger.c

```diff
@@ -95,8 +95,10 @@ static void gpio_callback(const struct device *dev,
 }
 
 #ifdef CONFIG_CCS811_TRIGGER_OWN_THREAD
-static void irq_thread(struct ccs811_data *drv_data)
+static void irq_thread(struct ccs811_data *dev)
 {
+	struct ccs811_data *drv_data = drv->data;
```

Comment:
this looks wrong

Reply:
Even when taking this change into consideration?

https://github.com/zephyrproject-rtos/zephyr/pull/46669/files#diff-294c2b826b94f89f6c60b84b58b1e804d1ea2dbc68c8ee2da7857560b4db7421L185-L189

Reply:
I mean `drv->data` should be `dev->data`

Reply:
Oh yes! Thank you. This should've been caught by the CI?

Reply:
`tests/drivers/build_all/sensor/prj.conf` doesn't seem to enable any trigger options, that's probably why